UKHomeOffice/sgmr-service
# SPCR-286 Change content for submitted voyage plan confirmation page

### AC / Description of changes
Update content of the submission confirmation page, according to the new prototype (attached screenshot). The content should not include the reference number.

<img width="681" alt="Screenshot 2022-04-21 at 09 59 26" src="https://user-images.githubusercontent.com/72464825/164419573-a99810f1-841a-41ff-a960-c715722d3d13.png">


### To Test
- Submit a voyage plan
- The content should match the screenshot
- The two external links should open in new tabs
- The 'clicking here' link should take you to the beginning of the voyage plan (help page)
- The continue to home page should take you to the dashboard

### Notes
The reference number will be added in another ticket
---
* remember to merge to feature branches not master *
